.github/workflows/validate-release-notes-pre-land.yml

```diff
@@ -24,7 +24,8 @@ concurrency:
 
 jobs:
   diff:
-    name: Determine PR eligibility for ${{ github.event.pull_request.number }}
+    name: Determine PR eligibility for 
```

Comment:
Should this be changed?

Reply:
weird, not sure why I deleted this. will update